NEWS.rst

```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.15 (2020-03-13)
+-----------------
+* Improve S3 chunk store: check tokens, improve timeouts and retries (#272 - #277)
```

Comment:
So you want to say anything specific about truncated reads?

Reply:
Good idea.